global.json

```diff
@@ -10,6 +10,7 @@
     "Microsoft.DotNet.Arcade.Sdk": "1.0.0-beta.19372.10",
     "Microsoft.DotNet.Helix.Sdk": "2.0.0-beta.19372.10",
     "Microsoft.Build.NoTargets": "1.0.53",
-    "Microsoft.Build.Traversal": "2.0.2"
+    "Microsoft.Build.Traversal": "2.0.2",
+    "Microsoft.NET.Sdk.IL": "3.0.0-preview1-27101-02"
```

Comment:
Where does this version come from? Is it updated automatically somehow? Doesn't look like it's part of Versions.props for example.

Reply:
I looked in dotnet/corefx and it seems that `Microsoft.NET.Sdk.IL` can be update automatically by dotnet-maestro bot (e.g. https://github.com/dotnet/corefx/commit/739f4437298d3d010f3427f557bf6d189aac67a7). I will figure out how to do the same in dotnet/coreclr after I merge this change.